src/test/java/com/mongodb/kafka/connect/source/producer/SchemaAndValueProducerTest.java

```diff
@@ -409,7 +417,9 @@ static String generateJson(final boolean simplified) {
             + " \"documentKey\": %s,"
             + " \"updateDescription\":"
             + " {\"updatedFields\": %s,"
-            + " \"removedFields\": [\"legacyUUID\"]},"
+            + " \"removedFields\": [\"legacyUUID\"],"
+            + " \"truncatedArrays\": [%s],"
+            + " \"disambiguatedPaths\": {\"home.town\": \"New York City\"}},"
```

Comment:
I believe `disambiguatedPaths` is an object with strings as keys and arrays as values. From the docs:
```
"disambiguatedPaths": {
   "home.town": [ "home.town" ],
   "residences.0.0": [ "residences", 0, "0" ]
}
```